applications/workspaces/backend/api/migrations/0008_auto_20220524_1006.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.11 on 2022-04-28 13:08
+
+from django.db import migrations
+
+from api.models import Population, PopulationStatus
+from api.services.population_service import split_cells_per_segment
+
+
+def update_existent_populations(apps, schema_editor):
```

Comment:
Adds similar content to the emptied migrations. Generate images, splits csv's and updates status of existent populations